security-indicators/base/argo-workflows/aggregator-template.yaml

```diff
@@ -87,15 +87,15 @@ spec:
                 name: argo-artifact-repository-secrets
                 key: secretKey
 
-          - name: clocout
-            path: /mnt/workdir/cloc_results
+          - name: aggreggatedout
```

Comment:
I just always struggle reading these :)

Reply:
You are right, sorry :) I will add `_` in another PR :)